Library/Formula/gedit.rb

```diff
@@ -0,0 +1,180 @@
+class Gedit < Formula
+  desc "The GNOME text editor"
+  homepage "https://wiki.gnome.org/Apps/Gedit"
+  url "https://download.gnome.org/sources/gedit/3.16/gedit-3.16.2.tar.xz"
+  sha256 "db60a9408711b00f3780f5f5b657e9f44722c88c683ba6e9f2b4885631f429aa"
+
+  depends_on "pkg-config" => :build
+  depends_on "intltool" => :build
+  depends_on "itstool" => :build
+  depends_on "gtk+3"
+  depends_on "gtk-mac-integration"
+  depends_on "gobject-introspection"
+  depends_on "enchant"
+  depends_on "iso-codes"
+  depends_on "libxml2"
+  depends_on "libpeas"
+  depends_on "gtksourceview3"
+  depends_on "gsettings-desktop-schemas"
+  depends_on "gnome-icon-theme"
+
+  # ensures that gtk-mac-integration has been linked against gtk+3
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!